src/main/java/org/folio/service/request/RequestBatchResourceService.java

```diff
@@ -0,0 +1,91 @@
+package org.folio.service.request;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.folio.rest.jaxrs.model.Request;
+import org.folio.service.BatchResourceService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.rits.cloning.Cloner;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+
+public class RequestBatchResourceService {
+  private static final Logger LOG = LoggerFactory.getLogger(RequestBatchResourceService.class);
+
+  private final BatchResourceService batchResourceService;
+  private final Cloner cloner;
```

Comment:
Why does the cloner need to be a field? It appears it is only used within the `cloneRequest` method.

Reply:
Id does not have a state, so the same instance can be used to clone all the requests in a batch package.

Reply:
Is it particularly expensive to instantiate a cloner? I'm asking because separating it into a field that needs initialising in the constructor means knowing about 3 places rather than only one.